test/helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var expect = require('must');
+var STANDARD_DELAY = 200; // 200ms between nested calls to the API so things settle
+
+function assertSuccessful(done, cb) {
```

Comment:
It's to keep the tests dry. We (almost) always asserted 
```
expect(err).not.to.exist();
expect(res).to.exist();
expect(xhr).to.be.an.object();
```
So I captured that here, along with the common logic for handling errors. If you have more assertions that the three above then you simply pass a callback to this helper and it will be executed as well.